debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/JextractCommand.java

```diff
@@ -44,4 +44,4 @@ public void run(String command, String[] args, Context context, PrintStream out)
 		}
 	}
 
-}
+}
```

Comment:
We want to keep the final newline.
The only other changes I see are removal of trailing whitespace.
Please restore the final newline or just revert changes to this file.